### ISSUE TITLE
OSX DMG Misses ANTLR4

### ISSUE THREAD

**Issue by trisyoungs** [disorderedmaterials/dissolve#1072]
**Describe the bug**
Packaging for OSX is broken, since the ANTLR4 runtime library is not included in the DMG file.

The CI logs show that the search path for the runtime is not the (correct) conan location:
```
ERROR: no file at "/usr/lib/libantlr4-runtime.4.9.3.dylib"
```

**Comment by trisyoungs** [comment]
Snippet from the error log dumped on OSX:
```
Dyld Error Message:
dyld: Using shared cache: DC9219E8-D54C-369D-81BB-CF54BFB2F6CD
Library not loaded: libantlr4-runtime.4.9.3.dylib
Referenced from: /Applications/Dissolve-GUI 3.app/Contents/MacOS/dissolve-gui
Reason: image not found
```

**Comment by trisyoungs** [comment]
Closed by #1079.
